test/jit/test_upgraders.py

```diff
@@ -131,6 +138,38 @@ def test_func():
         version = self._load_model_version(loaded_func)
         self.assertTrue(version == 4)
 
+    @unittest.skipIf(not _is_upgraders_enabled(), "Skipping because upgraders are not enabled")
+    def test_aten_linspace(self):
+        model_path = pytorch_test_dir + "/jit/fixtures/test_versioned_linspace_v7.ptl"
+        loaded_model = torch.jit.load(model_path)
+        sample_inputs = ((3, 10), (-10, 10), (4.0, 6.0), (3 + 4j, 4 + 5j))
+        for (a, b) in sample_inputs:
+            output_with_step, output_without_step = loaded_model(a, b)
+            # when no step is given, should have used 100
+            self.assertTrue(output_without_step.size(dim=0) == 100)
+            self.assertTrue(output_with_step.size(dim=0) == 5)
```

Comment:
It's a bit tedious, but it's still better to use variables here, like `expected_output = 100`, self.assertTrue(actual_output, expected_output)`.